src/librustc/lint/builtin.rs

```diff
@@ -242,7 +242,7 @@ declare_lint! {
 
 declare_lint! {
     pub TYVAR_BEHIND_RAW_POINTER,
-    Warn,
+    Deny,
```

Comment:
`Forbid` is better for testing existing lints, someone could `allow` it already.